package.json

```diff
@@ -3,8 +3,14 @@
   "version": "0.1.0",
   "description": "An express server to expose VROOM as a web API.",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node src/index.js"
+    "test": "npm run test:other && npm run test:js",
+    "start": "node src/index.js",
+    "fix:other": "npm run prettier --write",
+    "fix:js": "npm run test:js --fix",
+    "fix": "npm run fix:js && npm run fix:other",
+    "prettier": "prettier \"**/*.{json,md,scss,yaml,yml}\"",
+    "test:other": "npm run prettier --list-different",
+    "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.{js,jsx}\""
   },
```

Comment:
In my tests, running `npm run test:js` reports errors/warnings from eslint but running `npm run fix:js` does not fixes them. Maybe the `--fix` option is not passed along?

Reply:
Now it is, ty!